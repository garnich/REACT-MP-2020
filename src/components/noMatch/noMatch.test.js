import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NoMoatch from './noMatch';

Enzyme.configure({ adapter: new Adapter() });


describe('Test NoMoatch component', () => {
  it('<NoMoatch> test snapshot', () => {
    const wrapper = shallow(<NoMoatch />)

    expect(wrapper).toMatchSnapshot()
  })
});