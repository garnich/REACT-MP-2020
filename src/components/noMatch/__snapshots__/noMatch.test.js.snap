// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NoMoatch component <NoMoatch> test snapshot 1`] = `
<div
  className="wrapper-nomatch px-4 py-3"
>
  <Logo />
  <div
    className="content"
  >
    <p>
      Page not found
    </p>
    <p>
      404
    </p>
    <button>
      <Link
        to="/"
      >
        Go back to home
      </Link>
    </button>
  </div>
</div>
`;
